### ISSUE TITLE
GTA to Cityscapes translation

### ISSUE THREAD

**Issue by liortalker** [taesungp/contrastive-unpaired-translation#16]
Hi,

The results for GTA to Cityscapes translation are great!
Can you please explain what is the setting?
Is it the same as the other datasets?
I'm wondering because you seem to get the same 3D structure, with only the texture and lighting different, which is exactly what I want.

Can you clarify the settings? (which hyper parameters, models, etc)

**Comment by taesungp** [comment]
We used the following hyperparameters. 

`--batch_size 1 --preprocess scale_width_and_crop --load_size 1024 --crop_size 512`.

**Comment by Turlan** [comment]
The n_epochs and n_epochs_decay are still 200?

**Comment by taesungp** [comment]
We trained up to 80 epochs (40 + 40 decaying epochs), but I remember the results converged before that. I think 10 + 10 decaying epochs is enough.

**Comment by sparshgarg23** [comment]
Hello,apart from the above settings should the dataset for gta and cityscape be organized as described in docs/dataset.md
that is obtain GTA dataset and cityscape organize them as mentioned in docs/dataset.md
And once that's done run
```
python datasets/combine_A_and_B.py --fold_A /path/to/data/A --fold_B /path/to/data/B --fold_AB /path/to/data
```

**Comment by NO2-yh** [comment]
@taesungp Thanks for such amazing job！I have tried to reproduce GTA -> Cityscapes performance recently, but my results are really bad，could you please provide more details about training GTA to Cityscapes？（lr setting, loss can be decreased to which level?). Is it possible that the GTA -> Cityscapes pretrain model can be released?

**Comment by manuelmaior29** [comment]
> @taesungp Thanks for such amazing job！I have tried to reproduce GTA -> Cityscapes performance recently, but my results are really bad，could you please provide more details about training GTA to Cityscapes？（lr setting, loss can be decreased to which level?). Is it possible that the GTA -> Cityscapes pretrain model can be released?

I am also trying to use CUT for synthetic-to-real domain adaptation, so some details about the training setup used on GTA -> Cityscape would be really useful for me too
